src/connector.js

```diff
@@ -0,0 +1,769 @@
+				}
+			);
+
+			let resultLink = sectionNode.querySelector( ".result-link" );
+			resultLink.onclick = () => { interactiveResult.select(); };
+			resultLink.oncontextmenu = () => { interactiveResult.select(); };
+			resultLink.onmousedown = () => { interactiveResult.select(); };
+			resultLink.onmouseup = () => { interactiveResult.select(); };
+			resultLink.ontouchstart = () => { interactiveResult.beginDelayedSelect(); };
+			resultLink.ontouchend = () => { interactiveResult.cancelPendingSelect(); };
+
+			resultListElement.appendChild( sectionNode );
+		} );
+	}
+	else if ( resultListState.firstSearchExecuted && !resultListState.hasResults && !resultListState.hasError ) {
+		resultListElement.innerHTML = noResultTemplateHTML;
+	}
+}
+
+// Update heading that has number of results displayed
+function updateQuerySummaryState( newState ) {
+	querySummaryState = newState;
+	
+	if ( !querySummaryElement ) {
+		return;
+	}
+	
+	if ( resultListState.firstSearchExecuted && !querySummaryState.isLoading && !querySummaryState.hasError ) {
+		querySummaryElement.textContent = "";
+		if ( querySummaryState.total > 0 ) {
+			let numberOfResults = querySummaryState.total.toLocaleString();
+
+			querySummaryElement.innerHTML = ( ( querySummaryState.query !== "" && !params.isAdvancedSearch ) ? querySummaryTemplateHTML : noQuerySummaryTemplateHTML )
+				.replace( '%[numberOfResults]', numberOfResults )
+				.replace( '%[query]', querySummaryState.query );
+		}
+	}
+	else if ( querySummaryState.hasError ) {
+		querySummaryElement.textContent = "";
+		querySummaryElement.innerHTML = resultErrorTemplateHTML;
+	}
+}
+
+// update did you mean
+function updateDidYouMeanState( newState ) {
+	didYouMeanState = newState;
+	
+	if ( !didYouMeanElement )
+		return;
+	
+	if ( resultListState.firstSearchExecuted ) {
+		didYouMeanElement.textContent = "";
+		if ( didYouMeanState.hasQueryCorrection ) {
+			didYouMeanElement.innerHTML = didYouMeanTemplateHTML.replace( '%[correctedQuery]', didYouMeanState.queryCorrection.correctedQuery );
+			const buttonNode = didYouMeanElement.querySelector( 'button' );
+			buttonNode.onclick = ( e ) => { 
+				updateSearchBoxFromState = true;
+				didYouMeanController.applyCorrection();
+				scrollToTop();
+				e.preventDefault();
+			};
+		}
+	}
+}
+
+// Update pagination
+function updatePagerState( newState ) {
+	pagerState = newState;
+	pagerElement.textContent = "";
+
+	if ( pagerState.hasPreviousPage ) {
+		const liNode = document.createElement( "li" );
+		liNode.innerHTML = previousPageTemplateHTML;
+		const buttonNode = liNode.querySelector( 'button' );
+		buttonNode.onclick = () => { 
+			pagerController.previousPage();
+			scrollToTop();
+		};
+		pagerElement.appendChild( liNode );
+	}
+	pagerState.currentPages.forEach( ( page ) => {
+		const liNode = document.createElement( "li" );
+		if ( page === pagerState.currentPage ) {
+			liNode.setAttribute( "class","active" );
+			liNode.setAttribute( "disabled","true" );
```

Comment:
A list item don't have an attribute named "disabled", but the inner button do.

```suggestion
			liNode.classList.add( "active" );
```

FYI - to disable the button state, you can do something like: `buttonNode.disabled = true`